corehq/apps/analytics/views.py

```diff
@@ -31,7 +31,7 @@ def post(self, request, *args, **kwargs):
         digester = hmac.new(settings.GREENHOUSE_API_KEY, request.body, hashlib.sha256)
         calculated_signature = digester.hexdigest()
 
-        if str(calculated_signature) == str(request.META.get('HTTP_SIGNATURE', '')):
+        if str(calculated_signature) == str(request.META.get('HTTP_SIGNATURE'.split()[1], '')):
```

Comment:
This is splitting the string `'HTTP_SIGNATURE'`, which I'm pretty sure is not what you want and will result in an index error due to the `[1]`.

It seems possible that the value of the `HTTP_SIGNATURE` header (?) may not have two parts when split, which means `[1]` may throw an index error. What do you think of making that code more fault-tolerant? (probably can't do it in one line and keep it readable)